### PR TITLE
Create POST operation to create a history

### PR DESCRIPTION
**Pull request by KevinQAstillo**

### CODE REVIEW

File: src/test/java/com/at/globalclasses/MongoDBConnection.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public boolean compareID(String collection, String id) {
     public String getObjectByID(String collection, String id) {
         String mObject = "";
         MongoCollection<Document> coll = mDataBase.getCollection(collection);
-        FindIterable<Document> findIterable = coll.find(Filters.eq("_id", new ObjectId(id)));
+        FindIterable<Document> findIterable = coll.find(Filters.eq("_id", new ObjectId(id))).projection(fields(exclude("_class"), excludeId()));
```
Comment (ramiroAT) [inline src/test/java/com/at/globalclasses/MongoDBConnection.java]: Do you think is posible passing the projection field as one parameter of the function?

**Review by ramiroAT — APPROVED** [review]

File: src/test/java/com/at/globalclasses/MongoDBConnection.java (hunk 1 above)
Comment (KevinQAstillo) [reply src/test/java/com/at/globalclasses/MongoDBConnection.java]: Yes, even In the US_030 is already implemented in a specific class

Comment (KevinQAstillo) [reply src/test/java/com/at/globalclasses/MongoDBConnection.java]: By the moment I can delete this change and keep  the original projection.

Comment (ramiroAT) [reply src/test/java/com/at/globalclasses/MongoDBConnection.java]: It's ok, I was wondering  if for the future we can add a method overloading and allow two different signatures for getObjectByID method. One with projection and other one without it.

**Review by IorchAT — APPROVED** [review]

**Pull request merged**